NEWS.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.0 - Released
```

Comment:
Please see https://wiki.folio.org/display/FOLIJET/Acquisitions+-+Release+Procedures#Acquisitions-ReleaseProcedures-BestPractices

Stories should be in descending order, and a link to the diff/changlog should be provided (even if it won't be usable until the tag is actually created)

Reply:
Also note the best practices section for naming of the release PR...

Reply:
My Bad I missed the Best Practices section. Just looked at the Procedure. Done now